pallets/ah-migrator/src/staking/bags_list.rs

```diff
@@ -194,19 +194,19 @@ impl<T: Config> crate::types::AhMigrationCheck for BagsListMigrator<T> {
 
 		// Assert storage "VoterList::ListBags::ah_post::length"
 		// Assert storage "VoterList::ListBags::ah_post::length"
-		assert_eq!(
+		/*assert_eq!(
```

Comment:
This doesn't seem good to me. We will have to investigate. This fails on any 4 recent snapshots that I can get from the CI?